.github/workflows/tail-call.yml

```
@@ -0,0 +1,114 @@
+name: Tail calling interpreter
+on:
+  pull_request:
+    paths:
+      - 'Python/bytecodes.c'
+      - 'Python/ceval.c'
+      - 'Python/ceval_macros.h'
+      - 'Python/generated_tail_call_handlers.c.h'
+  push:
+    paths:
+      - 'Python/bytecodes.c'
+      - 'Python/ceval.c'
+      - 'Python/ceval_macros.h'
+      - 'Python/generated_tail_call_handlers.c.h'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
```

Add colour to CI logs (added to the other files in https://github.com/python/cpython/pull/129196):
```suggestion
env:
  FORCE_COLOR: 1

jobs:
```